npm/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@ologbonowiwi/tailcall",
+  "version": "0.1.0",
+  "description": "",
```

Comment:
should be taken from cargo.toml.

Reply:
can I copy/paste or do it dynamically via a node file?